python/src/main/api.py

```diff
@@ -51,9 +63,35 @@ def get(self, email):
             return jsonify({'person': create_person(person).serialize()})
 
 
-# @api.route('/api/v1/items/<string:email>')
-# class Item(Resource):
-#     def put(self, ):
+@api.route('/api/v1/items/<string:person_email>/<string:name>/<string:price>/<int:priority>/<string:hyperlink>')
+class Items(Resource):
+     def put(self, person_email, name, price, priority, hyperlink):
```

Comment:
Do we have anything that is stopping a user putting in the same item multiple times?